mifosng-android/src/main/res/layout/item_individual_collection_sheet.xml

```diff
@@ -2,92 +2,120 @@
 <androidx.cardview.widget.CardView
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/cv_client"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@color/white"
-    android:foreground="?android:attr/selectableItemBackground"
     android:clickable="true"
-    app:cardUseCompatPadding="true" >
-<LinearLayout xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="horizontal"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:elevation="8dp">
+    app:cardUseCompatPadding="true"
+    app:cardCornerRadius="10dp"
+    android:layout_marginEnd="8dp"
+    android:layout_marginRight="8dp"
+    android:layout_marginTop="8dp"
+    android:layout_marginStart="8dp"
+    android:layout_marginLeft="8dp">
+
+    <LinearLayout
+        android:orientation="horizontal"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <com.mifos.mifosxdroid.views.CircularImageView
+            android:id="@+id/iv_user_picture"
+            android:layout_width="60dp"
+            android:layout_height="60dp"
+            android:layout_gravity="center_vertical"
+            android:elevation="2dp"
+            android:src="@drawable/ic_dp_placeholder"
+            app:border="false"
+            android:layout_marginLeft="@dimen/layout_padding_16dp"
+            android:layout_marginStart="@dimen/layout_padding_16dp"
+            android:layout_marginTop="@dimen/layout_padding_16dp"
+            android:layout_marginBottom="@dimen/layout_padding_16dp" />
```

Comment:
```suggestion 
 android:layout_margin="@dimen/layout_padding_16dp/>"
```
@danishjamal104  If possible please reduce it to only single line attribute `layout_margin` that might reduce redundancies and save 3 more lines

Reply:
If we use `margin` as a single attribute then it will apply a margin to all 4 sides but I need to apply a margin to all sides except the `right or end` that's why I defined all three.